addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4291,21 +4292,7 @@ export class OdooEditor extends EventTarget {
         const node = ev.target;
         // handle checkbox lists
         if (node.tagName == 'LI' && getListMode(node.parentElement) == 'CL') {
```

Comment:
```suggestion
        if (node.tagName == 'LI' && getListMode(node.parentElement) == 'CL' && isMouseInsideCheckbox(ev)) {
```
then remove the nested `if` and outdent its contents.

Reply:
done!